routing/turns_generator.cpp

```diff
@@ -668,12 +668,17 @@ size_t CheckUTurnOnRoute(TUnpackedPathSegments const & segments,
   auto const & masterSegment = segments[currentSegment - 1];
   if (masterSegment.m_path.size() < 2)
     return 0;
+  if (segments[currentSegment].m_path.size() < 2)
```

Comment:
This check will be duplicated on the first iteration of the loop below. Do we need it here?